docs/examples/academy/29. Working with Aggregated Cargo Movements Data.ipynb

```diff
@@ -247,39 +221,32 @@
       "text/plain": [
-       "<Figure size 432x432 with 1 Axes>"
+       "<Figure size 600x600 with 1 Axes>"
       ]
      },
      "metadata": {},
      "output_type": "display_data"
     }
    ],
    "source": [
-    "quantity_by_destination = cms.groupby('unloading_country').sum()[['quantity']]\n",
+    "quantity_by_destination = cms.groupby('unloading_country').sum(numeric_only=True)[['quantity']]\n",
```

Comment:
This is the only meaningful change in this file - it fixes an issue where the `.sum` method was not working after a Pandas upgrade.